### PR TITLE
Initialize the work for vsphere1

### PR DESCRIPTION
**Pull request by hongkailiu**
https://issues.redhat.com/browse/DPTP-1292

/cc @stevekuznetsov @alvaroaleman @jcpowermac

### CODE REVIEW

**Comment by alvaroaleman** [comment]
Can we rename the thing to something more meaningful, like `vsphere`?

File: clusters/JoinBuildFarm.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+To run test on an OpenShift cluster which is not managed by DPTP, the cluster has to join the CI build farm.
+
+## Claim a folder 
+
+Claim a folder in [README.md](../README.md), e.g.,
+
+```
+* [build-clusters](./build-clusters)
+    ...
+    * [03_cluster](./build-clusters/03_cluster): build cluster hosted on OpenStack managed by SPLAT-team.
```
Comment (jcpowermac) [inline clusters/JoinBuildFarm.md:12]:
```suggestion
    * [03_cluster](./build-clusters/03_cluster): build cluster hosted on vSphere managed by SPLAT-team.
```

**Comment by jcpowermac** [comment]
cc: @patrickdillon @abhinavdahiya

File: clusters/JoinBuildFarm.md (hunk 1 above)
Comment (hongkailiu) [reply clusters/JoinBuildFarm.md:12]: Updated.

**Comment by hongkailiu** [comment]
> Can we rename the thing to something more meaningful, like `vsphere`?

You mean the folder name, `build03` to `vsphere`?
What if another cluster on vsphere comes?

**Comment by alvaroaleman** [comment]
> You mean the folder name, build03 to vsphere?

That and later on the name of the kubeconfigs

> What if another cluster on vsphere comes?

Then we will name it `numberOfAlreadyExistingVsphereClusters`, e.G. `vsphere1`. Alternatively anything else that makes sense (e.G. `vsphere-$location`, `vsphere-$version` etc).

**Comment by hongkailiu** [comment]
@alvaroaleman 

Renamed to `vsphere01`.

**Comment by hongkailiu** [comment]
@jcpowermac

Is the cluster on vsphere ready to join?

**Comment by alvaroaleman** [comment]
> Renamed to vsphere01.

Can we please always avoid zero-prefixed numbers whenever possible? It just results in ppl building shitty parsing that assumes the number will always have to digits. Just use `vsphere1`.

**Comment by stevekuznetsov** [comment]
All good comments, Alvaro, but they break the existing pattern ...

**Comment by alvaroaleman** [comment]
> All good comments, Alvaro, but they break the existing pattern ...

By that logic we will keep on doing bad things just because we did that once in the past :upside_down_face:

**Comment by jcpowermac** [comment]
> @jcpowermac
> 
> Is the cluster on vsphere ready to join?

@hongkailiu Yes, it should be.

**Comment by hongkailiu** [comment]
@alvaroaleman and @stevekuznetsov 

Is it OK with the current folder name?
I can proceed with @jcpowermac after merge.

**Comment by alvaroaleman** [comment]
folder name SGTM

**Pull request merged**